dockerfiles/example/examples/c/Makefile

```diff
@@ -0,0 +1,13 @@
+SRCS = $(wildcard *.c)
```

Comment:
Wouldn't it make more sense to say that the docker context is always the repo root? Copying the files here is quite bad